cpp/memilio/epidemiology/state_age_function.h

```diff
@@ -365,6 +427,22 @@ struct ConstantFunction : public StateAgeFunction {
         return m_support_max;
     }
 
+    /**
+     * @brief Computes the mean value of the function. 
+     * 
+     * For ConstantFunction, the mean value is the function parameter.
+     *
+     * @param[in] dt Time step size used for the numerical integration (unused for ConstantFunction). 
+     * @param[in] tol The maximum support used for numerical integration is calculated using this tolerance (unused for ConstantFunction). 
+     * @return ScalarType mean value.
+     */
+    ScalarType get_mean(ScalarType dt = 1., ScalarType tol = 1e-10) override
```

Comment:
I don't think we should implement a get_mean method for a StateAgeFunction of type b). 
At the moment, the get_mean() method is only called for StateAgeFunctions of type a) that are used as TransitonDistributions. If we call the get_mean method for a SAF of type b) something went wrong (at least for the current state of the IDE model). This is why I would argue to not implement a get_mean method here and log an error, even if it is really easy to compute the mean.